keentools/geotracker/callbacks.py

```diff
@@ -198,11 +194,14 @@ def update_camobj(geotracker, context: Any) -> None:
         warn('INVOKE_DEFAULT', msg=ErrorType.CustomMessage,
              msg_content=msg)
 
-    if check_unbreak_rotaion_is_needed(geotracker.camobj):
-        msg = f'Camera {_unbreak_rotation_is_needed_warning}'
-        warn = get_operator(Config.kt_warning_idname)
-        warn('INVOKE_DEFAULT', msg=ErrorType.CustomMessage,
-             msg_content=msg)
+    prefs = settings.preferences()
+    if prefs.gt_auto_unbreak_rotation and \
+            check_unbreak_rotaion_is_needed(geotracker.camobj):
+        _log.info(f'Applying Unbreak Rotation to '
+                  f'{geotracker.camobj.name if geotracker.camobj else geotracker.camobj}')
```

Comment:
if geotracker.camobj is None than it would print 'Applying Unbreak Rotation to None'.
Probably better use constant string in else statement like 'GeoTracker camera'